.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Uses the same pattern rules for gitignore files
+# https://git-scm.com/docs/gitignore#_pattern_format
+# For more info about CODEOWNERS, visit
+# https://help.github.com/articles/about-codeowners/
+
+#
+# These are the default owners for everything in the repo.
+#
+* @anselmbradford @cfarm @contolini @jimmynotjim @Scotchester @sebworks
+
+# To add owners of a specific file pattern or directory, prepend the line with
+# the type to match. The last matching pattern takes the most precedence. Ex.
```

Comment:
I'd still really like to see this say "pattern" instead of "type", since it can be based on things other than file type.

Reply:
Ha, I thought you meant the usage of "type" on L:11, didn't even see L:12. I thought it was odd but just went with it, realizing now it makes a lot more sense this way.